docs/source/dev_guide/release.rst

```diff
@@ -81,7 +81,7 @@ Releasing Anaconda
 ------------------
 
 1. Be sure to have already completed :ref:`Releasing On GitHub <github-release>`. This triggers the CI/CD workflow that handles Anaconda releases.
-2. Wait until the CI/CD build is successful. You can view all workflows at `All Workflows <https://github.com/E3SM-Project/zppy/actions>`_.
+2. Wait for a bot PR to come up automatically on conda-forge after the GitHub release.
 3. Check the https://anaconda.org/e3sm/zppy page to view the newly updated package.
 
    * Release candidates are assigned the ``e3sm_dev`` label
```

Comment:
> Do you still want to do this on the conda-forge side

I think this is probably a good idea, so we can differentiate between release candidates and actual releases. I'm sure that will be helpful to @xylar when assembling Unified as well.

Reply:
Yes, we use the `e3sm_dev` label on conda-forge just like on the `e3sm` channel.  I absolutely have to have a release candidate for testing E3SM-Unified because the `conda-forge/label/e3sm_dev` channel takes priority over the `conda-forge` channel, meaning that it won't find a release for any package that has ever had a release candidate on the `e3sm_dev` label.  That's kind of inconvenient but I don't think there's a good way around it.

Reply:
> Here's a link on this (if you're still interested in keeping up with this): https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

I'm not quite sure I follow. Does the bot PR not work -- e.g., we have to make a manual PR with the `dev` or `rc` label?

Reply:
No, you have to do a release candidate manually on conda-forge.  `alpha`, `beta` and `rc` tags are ignored by the bot.

Reply:
You have to make the `rc` tag locally (I wouldn't make a release because an `rc` is *not* a release).  Then, you make a PR to conda-forge from your fork of the feedstock.  You need to start from the current `dev` branch and update the version number and the sha256 sum manually.  You need to set the build number back to 0 if needed.  You need to make the `dev` branch the target of the PR.  That way, the package build on conda-forge will end up with the `e3sm_dev` label.

Reply:
Yes, locally means here.

I really wouldn't do a release page for release candidates.  It's confusing when folks are looking for actual releases.